lib/Backend/SccLiveness.cpp

```diff
@@ -781,30 +781,36 @@ SCCLiveness::FoldIndir(IR::Instr *instr, IR::Opnd *opnd)
         {
             return false;
         }
-        indir->SetOffset(offset);
+        int32 newOffset = 0;
+        if(!Int32Math::Add(offset,indir->GetOffset(), &newOffset))
```

Comment:
i agree, this makes no sense

---
In reply to: [116345511](https://github.com/Microsoft/ChakraCore/pull/2849#discussion_r116345511) [](ancestors = 116345511)